chart/templates/_helpers.yaml

```diff
@@ -98,6 +98,13 @@ If release name contains chart name it will be used as a full name.
         name: {{ template "webserver_secret_key_secret" . }}
         key: webserver-secret-key
   {{- end }}
+  {{- if .Values.enableBuiltInSecretEnvVars.AIRFLOW__API_AUTH__JWT_SECRET }}
+  - name: AIRFLOW__API_AUTH__JWT_SECRET
```

Comment:
We only need to add this for Airflow 3 deployments. But, we have a bit of challenge in that, with the preinstall hook approach, the secret may not exist during upgrades...

Reply:
The `lookup` function also isn't a great answer here either... hmm, tricky.

Reply:
cc: @jedcunningham ?